application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -204,12 +181,12 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
 
     private static void handleSubcommand(@NotNull SlashCommandEvent event,
             @NotNull VoiceChannel voiceChannel, @NotNull String applicationId,
-            @Nullable Integer maxUses, @Nullable Integer maxAge, @NotNull String applicationName) {
+            @Nullable Long maxUses, @Nullable Long maxAge, @NotNull String applicationName) {
 
         voiceChannel.createInvite()
             .setTargetApplication(applicationId)
-            .setMaxUses(maxUses)
-            .setMaxAge(maxAge)
+            .setMaxUses(Math.toIntExact(maxUses))
+            .setMaxAge(Math.toIntExact(maxAge))
```

Comment:
Why use a long if we want an int?
Currently the flow is like:

Int -> long -> int

Why not just keep it an int?